content/blog/2017-11-10-rprofile-mara-averick.md

```diff
@@ -0,0 +1,132 @@
+---
+slug: rprofile-mara-averick
+title: ".rprofile: Mara Averick"
+date: 2017-11-10
+authors:
+  - name: Kelly O'Briant
+    url: https://kellobri.github.io/
+categories: blog
+topicid: 921
+tags:
+- R
+- community
+- interviews
+---
+
+<div class="alert alert-info" role="alert">
+In this occasional series, we interview someone using a loosely defined set of interview questions for the purpose of “demystifying the creative and development processes of R community members”. This interview was conducted and prepared by Kelly O'Briant as part of an rOpenSci unconf17 project.
+</div>
+
+<br>
+
+<img src="/assets/blog-images/2017-11-10-rprofile-mara-averick/mara_averick.png" alt="Mara Averick, Data Nerd At Large" align="left" style="margin: 0px 20px", width="250px">
```

Comment:
Right now this image does not exist. On the new blog, place this in: 

```
public/img/blog-images2017-11-10-rprofile-mara-averick/mara_averick.png
```

Do that and reference the image accordingly and it should work.

Reply:
@karthik I don't see a directory structure currently in place for public/img/blog-images. Am I correct in assuming the public directory you're referring to is on the same level as /content and /themes?

Reply:
Sorry here is the correct path:

```
themes/ropensci/static/img/blog-images
```
Create a folder for this post like you did in the old repo, and drop Mara's image there. Then reference it with just 

```/img/blog-images/...```